packages/jest-resolve/src/__tests__/resolve.test.ts

```diff
@@ -75,11 +75,11 @@ describe('isCoreModule', () => {
     expect(isCore).toBe(true);
   });
 
-  it('returns true if using `node:` URLs and `moduleName` is not a core module.', () => {
+  it('returns false if using `node:` URLs and `moduleName` is not a core module.', () => {
```

Comment:
I'm not sure if this is intended, and should we really support this usage?

Reply:
yeah, this should be `false` 👍